templates/humanagent/src/skills/waas.ts

```
@@ -0,0 +1,91 @@
+async function handleWallet(context: XMTPContext) {
+  const {
+    message: {
+      content: { skill, params },
+      sender,
+    },
+    walletService,
+  } = context;
+
+  switch (skill) {
+    case "create":
+      const created = await walletService.createWallet(sender.address);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)